dts/bindings/sensor/ti,tmp116.yaml

```diff
@@ -10,5 +10,5 @@ include: i2c-device.yaml
 properties:
     alert-gpios:
       type: compound
-      category: optional
```

Comment:
btw, the driver does not use any gpios, https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/sensor/tmp116/tmp116.c. Probably the property should be removed?

Reply:
[Looks like](http://www.ti.com/lit/ds/symlink/tmp116.pdf) there's an `ALERT` pin at least, but yeah, maybe best to remove it until it's used.

I'll send a separate PR for it to get comments. Could merge either one then.

Fixed up this one some more to also make `alert-gpios` a `phandle-array`, to match some other bindings.

Reply:
Alternative PR that removes it instead: https://github.com/zephyrproject-rtos/zephyr/pull/21605